api/internal/handler/route/route.go

```diff
@@ -218,7 +220,11 @@ func (h *Handler) List(c droplet.Context) (interface{}, error) {
 			if input.Label != "" && !utils.LabelContains(obj.(*entity.Route).Labels, labelMap) {
 				return false
 			}
-
+			
+		        status, _ := strconv.Atoi(input.Status)
+			if input.Status != "" && uint8(obj.(*entity.Route).Status) != uint8(status) {
```

Comment:
There is an easy way.Just compare `strconv.Itoa(int(obj.(*entity.Route).Status))` and status.

Reply:
Yes, but my biggest problem now is that no matter whether I use uint8 or string for comparison, my test cases can't pass. There's no problem with the function itself. Can you help me take a look at my test cases?  Thks

Reply:
@Jaycean you can try to `fmt.Sprintf("%d", obj.(*entity.Route).Status)` to compare in string type.